### PR TITLE
Example Config for Creality Ender3S1

### PR DESCRIPTION
**Pull request by BlackScript**
Signed-off-by: Patrick Hügel <p.huegel@arrowtec-media.de>

### CODE REVIEW

**Comment by Sineos** [comment]
Thanks for your contribution.

Some comments:

* Please cross check with https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md
  * the filename should contain the year of the printer release
  * signed-off line
* BL-Touch
  * Does this CR-Touch sensor work like a BL-Touch?
  * Why does it have no x / y offsets?

**Comment by BlackScript** [comment]
> Thanks for your contribution.
> 
> Some comments:
> 
> * Please cross check with https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md
>   
>   * the filename should contain the year of the printer release
>   * signed-off line
> * BL-Touch
>   
>   * Does this CR-Touch sensor work like a BL-Touch?
>   * Why does it have no x / y offsets?

Its only an example conf and the Main was to conrribute the Pins so other could help to fix Rest ;)

**Comment by Sineos** [comment]
> Its only an example conf and the Main was to conrribute the Pins so other could help to fix Rest ;)

Your contribution is appreciated. To keep a constant quality in the Klipper main-line only example configurations will be accepted that fulfill the requirements. Other users shall be enabled to use the configurations fully working out-of-the-box

**Comment by BlackScript** [comment]
> > Its only an example conf and the Main was to conrribute the Pins so other could help to fix Rest ;)
> 
> Your contribution is appreciated. To keep a constant quality in the Klipper main-line only example configurations will be accepted that fulfill the requirements. Other users shall be enabled to use the configurations fully working out-of-the-box

so. now its a working config as far as i can say .. any other what i need to modify to get it in the main branch ?

**Comment by BlackScript** [comment]
which more feedback is needed from me ?

**Comment by BlackScript** [comment]
Signed-off-by: Patrick Hügel
@mental405 everything okay with this now ?

File: config/printer-creality-ender3-s1-2021.cfg [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+horizontal_move_z: 5
+mesh_min: 15, 15
+mesh_max: 206.6,190
+probe_count: 5,5
+fade_start: 1
+fade_end: 10
+fade_target: 0
+mesh_pps: 2,2
+algorithm: bicubic
+bicubic_tension: 0.2
```
Comment (Sineos) [inline config/printer-creality-ender3-s1-2021.cfg:69]: Are these settings needed?
`fade_start`, `mesh_pps: 2,2`, `bicubic_tension: 0.2` are default anyway and as such should not be repeated in the config.
In general fading can lead dimensional errors in Z

Comment (BlackScript) [reply config/printer-creality-ender3-s1-2021.cfg:69]: generally i though it should be a real config and thats my config that i using. and if fading anyway enabled by default i could remove it here.

File: config/printer-creality-ender3-s1-2021.cfg [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+screw3: 238,232
+screw3_name: RR
+screw4: 62,232
+screw4_name: RL
+horizontal_move_z: 10
+speed: 80
+screw_thread: CW-M3
+
+[extruder]
+max_extrude_only_distance: 1000.0
```
[the remark this answers is not in the document]
Comment (BlackScript) [reply config/printer-creality-ender3-s1-2021.cfg:85]: it was for extruder calibration. so okay i remove it

File: config/printer-creality-ender3-s1-2021.cfg (hunk 1 above)
Comment (Sineos) [reply config/printer-creality-ender3-s1-2021.cfg]: > generally i though it should be a real config

The config should contain what is needed to get the printer working safely and with its default options. Fading is rather something that can be used according to personal preferences and models that are printed.
If not strictly needed for this type of printer, then please remove it.

File: config/printer-creality-ender3-s1-2021.cfg [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+z_offset: 0
+speed: 5.0
+stow_on_each_sample: false
+
+[bed_mesh]
+speed: 120
+mesh_min: 15, 15
+mesh_max: 206.6,190
+probe_count: 5,5
+fade_end: 10
```
Comment (Sineos) [inline config/printer-creality-ender3-s1-2021.cfg:63]: Please remove

**Review by Sineos — COMMENTED** [review]
One small leftover from the fade setting otherwise LGTM.
Many thanks

**Comment by Sineos** [comment]
Looks good to me.
Some formal aspect, if you be so kind:
* Signed-off lines needs to be formatted like `Signed-off-by: Full Name <email@address.xy>` (preferably edit your initial post)
* Add the doc ref to the top `# See docs/Config_Reference.md for a description of parameters.` after the build instructions
* Add the printer to the build test environment: https://github.com/Klipper3d/klipper/blob/6e6ad7b5201d3452aa605f4ae852c51239c2c7d8/test/klippy/printers.test#L142

Many thanks for your support. Appreciated 👍

**Comment by BlackScript** [comment]
> Looks good to me. Some formal aspect, if you be so kind:
> 
> * Signed-off lines needs to be formatted like `Signed-off-by: Full Name <email@address.xy>` (preferably edit your initial post)
> * Add the doc ref to the top `# See docs/Config_Reference.md for a description of parameters.` after the build instructions
> * Add the printer to the build test environment: https://github.com/Klipper3d/klipper/blob/6e6ad7b5201d3452aa605f4ae852c51239c2c7d8/test/klippy/printers.test#L142
> 
> Many thanks for your support. Appreciated 👍

everything done.

**Comment by BlackScript** [comment]
@Sineos Anything further ? or why the commit not proccesed ?

**Comment by BlackScript** [comment]
ITS Not ! inactive ... just the ppl here seems to be ... dont understand why it is so complicated to merge this simple config into the master ...

**Comment by BlackScript** [comment]
@Sineos could you mark your review as complete ? or did you need some further changes ? and probably remove the inactive label :)

Thx

**Comment by BlackScript** [comment]
push ... :-(

**Comment by BlackScript** [comment]
push

**Comment by dingyifei** [comment]
> push

you still need to change your commit messages etc

**Comment by BlackScript** [comment]
> > push
> 
> you still need to change your commit messages etc

i already did this all as i can see .. what did you mean is needed ?

**Comment by BlackScript** [comment]
@KevinOConnor probably the request just not attract your attention because the wrong labels ?! but i cant change them as far as i can see .. so i try this now this way because its a really long time for just integrate a config ? i would really thankful if you could accept the request ... Thank You !

**Comment by BlackScript** [comment]
hm ... :(

**Comment by BlackScript** [comment]
> 
Thanks for your reply Kevin. The Problem is that i don't understand whats wrong with the config ... i thought i had already done all 

> This PR would need to follow the guidelines at https://www.klipper3d.org/Example_Configs.html . At a minimum, points 7 and 7e. You can use `config/generic-bigtreetech-skr-mini-e3-v2.0.cfg` as an example of the preferred wording/ordering of a config file. I'm not sure what the intent of the extruder rotation_distance is, but I'm sure the printer doesn't have a hobbed bolt that is ~2mm in circumference.
> 
> It seems to me this is the 3rd time that the guidelines have been pointed out. We're expecting contributors to find and resolve issues on their own.
> 
> -Kevin

Hi kevin,  thanks for your reply .. The Problem is that i not really understood what exactly wrong in my config i already though i did changed all whats needed already. The rotation distance is just what i figured out after calibration and the printer now already works over months fine with this config :-( i now did change the file and hope this was the last thing what i not correctly understood from the template ...

**Comment by BlackScript** [comment]
@KevinOConnor how could i remove the pending feedback to show that i changed something ? or could i do this not by myself ?

**Comment by Fisheiyy** [comment]
> @KevinOConnor how could i remove the pending feedback to show that i changed something ? or could i do this not by myself ?

you cant do it by yourself, someone has to do it for you

**Comment by BlackScript** [comment]
> > @KevinOConnor how could i remove the pending feedback to show that i changed something ? or could i do this not by myself ?
> 
> you cant do it by yourself, someone has to do it for you

okay. then i going to wait now ...

**Pull request closed without merging**